File: android/Girlsfleet/app/src/main/java/com/circus/girlsfleet/IRenderer.java
package com.circus.girlsfleet;

import android.graphics.Color;
import android.graphics.Rect;

public interface IRenderer {

    void drawRect(Rect r);
    void drawRect(int l,int t,int w,int h);
    void fillRect(Rect r);
    void fillRect(int l,int t,int w,int h);
    void fillRect(Vector center,int radius);
    void drawLine(int x1,int y1,int x2,int y2);
    void drawText(int l,int t,String txt);
    void setColor(int c);
}

File: android/Circus/app/src/main/java/com/personal/circus/IRenderer.kt
package com.personal.circus;

import android.graphics.Bitmap
import android.graphics.Rect
import android.media.Image

interface IRenderer{

    fun drawRect(r: Rect);
    fun drawRect(x:Int,y:Int,w:Int,h:Int);
    fun fillRect(r:Rect);
    fun fillRect(x:Int,y:Int,w:Int,h:Int);
    fun fillRect(v:Vector,r:Int)
    fun drawLine(x1:Int, y1:Int, x2:Int, y2:Int);
    fun drawText(x:Int,y:Int,t:String);
    fun drawImage(x:Int,y:Int,im: Bitmap)
    fun calcTextRect(t:String):Rect;
    fun setColor(c:Int);
}